docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1065,6 +1065,16 @@ Valid values are `-1` (unlimited), `integer` (in kilobytes), `xMB` (in megabytes
 
 Default: `1GB`
 
+##### --yb_enable_upsert_mode
```

Comment:
I don't understand why we find this in this doc. And if it is a YSQL GUC it is not `--yb_enable_upsert_mode` but `yb_enable_upsert_mode=true`

Reply:
it was the first question to understand if this was about a Gflag or a GUC
besides that 
- "should be maintained" is not clear. iI they are maintained, no need to mention it. If they are not, it must be clear
-  if "batching" refers to the COPY rows per transaction, I don't see how it is related to upsert. If it is it must be clear that it relates to COPY batching because other batching (multi-statement inserts, multi-value inserts) work